Extensions/fresh_prince.js

```diff
@@ -11,6 +11,39 @@ XKit.extensions.fresh_prince = new Object({
 
 	running: false,
 
+	preferences: {
+		fresh_song_of_bel_air: {
+			text: "NOW THIS IS A STORY ALL ABOUT HOW MY LIFE",
+			default: true,
+			value: true
+		},
+		flavor: {
+			text: "GOT FLIPPED TURNED UPSIDE DOWN",
+			default: "/AVbQo3IOC_A",
```

Comment:
Tried that but for some reason that fucked with with the Ace Attorney of Bel Air option. It basically just ended adding the 3 instead of the full URL